packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## 1.3.0
+- New SuiteCloud: Validate Project command
```

Comment:
Not a big deal but maybe it's better to use quotes for command name to make it a bit more readable:
New "SuiteCloud: Validate Project" command